subcommittees/mentoring/roles/mentors.md

```diff
@@ -0,0 +1,15 @@
+# Mentors
+
+## Responsibilities
+
+
+## Time commitment
+
```

Comment:
~4 hours/month for 3-5 months (or longer if accepting new Mentees), this breaks down to about 2 hours prep, 1 hour meeting and 1 hour answering emails per month. Variable depending on meeting frequency and prep time.